src/test/java/edu/cornell/kfs/concur/batch/fixture/ConcurParameterConstantsFixture.java

```diff
@@ -0,0 +1,26 @@
+package edu.cornell.kfs.concur.batch.fixture;
+
+import java.util.HashMap;
+
+public class ConcurParameterConstantsFixture {
+
+    HashMap<String, String> definedParameters;        
+
+    public ConcurParameterConstantsFixture() {
+        this.definedParameters = new HashMap<String, String>();
+        this.definedParameters.put("CONCUR_AP_PDP_ORIGINATION_CODE", "Z6");
```

Comment:
I would recommend using the constants from ConcurParameterConstants for the keys.